teams/Aragon One/2019.md

```diff
@@ -156,19 +156,15 @@ While we commit to getting it to a beta state, the maintainance of it will have
 
 ### aragonOS
 
-#### 🔜 Global emergency failsafe
+#### ⏳ Global emergency failsafe
 
 Provide an opt-in failsafe for organizations and individual apps to safely freeze their state during times of emergency
 
 As a new system built on highly experimental technology, it is highly desirable to include failsafes in the event of security breaches or other emergency events. Organizations should be able to opt-in their entire organization or a particular set of applications to a failsafe that freezes their state and execution when triggered.
 
-[Related GitHub issue](https://github.com/aragon/aragonOS/pull/518)
-
-#### Decouple aragonPM implementation
+**Status**: Research and initial work done, further integration into the Aragon client has been removed from the 2019 scope.
```

Comment:
We have an initial implementation of the global emergency failsafe, but we don't have design nor frontend bandwidth to integrate it within the Aragon client during Q4 2019.